meteor/client/ui/Settings/ConfigManifestSettings.tsx

```diff
@@ -80,6 +81,65 @@ function filterLayerMappings(
 	return result
 }
 
+function getFilteredGfxDefaultTableValues<DBInterface extends { _id: ProtectedString<any> }>(
+	item: ConfigManifestEntryFilterDefaultsFromShowMapping<boolean>,
+	configPath: string,
+	object: DBInterface,
+	alternateObject?: any
+): string[] {
+	const sourceAttribute = `${configPath}.${item.sourceTableId}`
+	const targetAttribute = `${configPath}.${item.targetTableId}`
+	const sourceTable = objectPathGet(object, sourceAttribute) ?? objectPathGet(alternateObject, sourceAttribute)
+	const targetTable = objectPathGet(object, targetAttribute) ?? objectPathGet(alternateObject, targetAttribute)
+	const defaultRow = targetTable.find((row) => typeof row === 'object' && row[item.targetCompareColumnId] !== undefined)
+	const result: string[] = []
+	if (!Array.isArray(sourceTable) || !Array.isArray(targetTable) || !defaultRow) {
+		return result
+	}
+
+	sourceTable.forEach((row) => {
+		if (!(typeof row === 'object' && row[item.sourceCollectColumnId] !== undefined)) {
+			return
+		}
+		if (gfxSchemaIsUnavailable(row, item)) {
+			item.sourceCompareColumnId = 'GfxSetup'
+			item.targetCompareColumnId = 'DefaultSetupName'
+		}
+		filterAndCollectGfxDefaultValues(row, item, defaultRow[item.targetCompareColumnId], result)
+	})
+	result.push('N/A')
+	return result
```

Comment:
I am not a fan of how `result` is updated with side effects. If `filterAndCollectGfxDefaultValues` can be changed to returning a `string[]`, then this entire block can be changed to a filter-map sequence/reduce – making it easier to understand the flow.